### PR TITLE
Use DOM for variable watchers

### PR DESCRIPTION
**Pull request by tjvr**
For #252.

Comparison:
watchers-canvas
watchers-dom

### CODE REVIEW

File: phosphorus.js [hunk 1]
```diff
@@ -1949,121 +2041,7 @@ var P = (function() {
...
-      context.stroke();
-      context.fill();
-
-      context.fillStyle = '#fff';
-      context.textAlign = 'center';
-      context.fillText(value, dw / 2, dh - 5);
-    }
-
-    context.restore();
+    this.value.textContent = value + '\u200c';
```
Comment (nathan) [inline phosphorus.js]: There's a very specific reason I do the `\u200c` thing in Visual and it doesn't apply here.

Comment (tjvr) [inline phosphorus.js]: Without it, empty values get collapsed:

screen shot 2016-03-23 at 21 30 54

Comment (nathan) [inline phosphorus.js]: Not an appropriate fix.

Comment (tjvr) [inline phosphorus.js]: What would you suggest? :)

**Comment by tjvr** [comment]
@nathan Is this better?

**Pull request closed without merging**